includes/wikia/DefaultSettings.php

```diff
@@ -614,6 +614,7 @@
 include_once( "$IP/extensions/wikia/Thumbnails/Thumbnails.setup.php" );
 include_once( "$IP/extensions/wikia/InstantGlobals/InstantGlobals.setup.php" );
 include_once( "$IP/extensions/wikia/UserTools/UserTools.setup.php" );
+include_once( "$IP/extensions/wikia/SpecialPageViews/SpecialPageViews.setup.php" );
```

Comment:
Should not be enabled by default -- we should ask Trevor and Kirk what their rollout plans are.